shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/View.kt

```diff
@@ -21,10 +17,10 @@ data class Speakers(
 }
 
 @OptIn(ExperimentalResourceApi::class)
-enum class EventDay(val title: StringResource) {
-    May22(Res.string.day_1),
-    May23(Res.string.day_2),
-    May24(Res.string.day_3);
```

Comment:
Removing these hardcoded day strings for now. We should see if these can be computed more reasonably from the backend data.

Reply:
Sometimes formatting of days is changed in the sessionize for the website requirements, so we need to compute this from the actual sessions timestamps